server/container/cli-integration/src/test/java/org/apache/james/cli/SieveCommandsIntegrationTest.java

```diff
@@ -30,14 +31,13 @@
 import org.apache.james.modules.protocols.SieveProbeImpl;
 import org.apache.james.modules.server.JMXServerModule;
 import org.apache.james.sieverepository.api.exception.QuotaNotFoundException;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
You should not be using wildcards in imports

Reply:
what about providing our codestyle in a formalized way like a checkstyle file ?

Reply:
That would be nice. Maybe also a step on "How to disable wildcard import in IDEA". It keeps converting them to wildcards if the single class imports a certain number.